src/index.js

```diff
@@ -1,11 +1,11 @@
-import _ from 'lodash';
 import './style.css';
-import UI from './modules/func.js';
-import Score from './modules/class.js';
+import Score from "./modules/class.js";
```

Comment:
- [ ]  **I would advise you to fix this and submit it again because the App needs to run as expected. An unknown `class.js` file is causing confusion.**
![Capture](https://user-images.githubusercontent.com/63972670/216474863-62f60f6e-65db-4756-993a-b48386bf64ce.PNG)